### PR TITLE
docs(readme): ignore bots

### PR DESCRIPTION
**Pull request by Favna**
Y'know... when even I just blindly copy paste the code and make the mistake I'm sure there are many many other people who will also 😂😂

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ import config from './config.json';
 
 const client = new Client()
 	.on(ClientEvents.MessageCreate, async (message): Promise<void> => {
+        if (message.author.bot) return;
```
Comment (kyranet) [inline README.md:11]:
```suggestion
		if (message.author.bot) return;
```

Comment (kyranet) [reply README.md:11]: And while you're at it, can you change everything else to use tabs?

Comment (bdistin) [inline README.md]:
```suggestion
		if (message.author.bot) return;
```

Comment (kyranet) [reply README.md]: 🐢

**Pull request merged**